api/app/services/spree/webhooks.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
+  module Webhooks
+    def self.disable_webhooks(&block)
+      webhooks_disabled_previously = ENV['DISABLE_SPREE_WEBHOOKS']
+      begin
+        ENV['DISABLE_SPREE_WEBHOOKS'] = 'true'
+        block.call
```

Comment:
You could use `yield` instead of `block.call` here, proven to be a bit faster and you get rid of specifying a parameter.